packages/react-components/src/theme-provider/src/ThemeProvider.css

```diff
@@ -42,23 +212,8 @@
     --primary-900: var(--overcast-900);
 }
 
-/* OVERCAST | DARK */
-.o-ui-overcast-theme.o-ui-dark-color-scheme {
-    --primary-50: black;
-    --primary-100: black;
-    --primary-200: black;
-    --primary-300: black;
-    --primary-400: black;
-    --primary-500: black;
-    --primary-600: black;
-    --primary-700: black;
-    --primary-800: black;
-    --primary-900: black;
-}
-
 /* DESKTOP */
-/* DESKTOP | LIGHT */
-.o-ui-desktop-theme.o-ui-light-color-scheme {
+.o-ui-desktop-theme {
```

Comment:
you shouldn't remove `.o-ui-light-color-scheme`. Without a color scheme it shouldn't work.

Reply:
Both were the same that's why, I will be more explicit in my declarations.